terraform/ecs/main.tf

```diff
@@ -257,17 +318,35 @@ module "validator_without_sample_app" {
   mocked_server_validating_url = "http://${aws_lb.mocked_server_lb.dns_name}:${module.common.mocked_server_lb_port}/check-data"
 
   ecs_cluster_name    = module.ecs_cluster.cluster_name
-  ecs_task_arn        = aws_ecs_task_definition.aoc.arn
-  ecs_taskdef_family  = aws_ecs_task_definition.aoc.family
-  ecs_taskdef_version = aws_ecs_task_definition.aoc.revision
+  ecs_task_arn        = aws_ecs_task_definition.aoc[0].arn
+  ecs_taskdef_family  = aws_ecs_task_definition.aoc[0].family
+  ecs_taskdef_version = aws_ecs_task_definition.aoc[0].revision
 
   aws_access_key_id     = var.aws_access_key_id
   aws_secret_access_key = var.aws_secret_access_key
 
   depends_on = [aws_ecs_service.aoc_without_sample_app]
 }
 
+module "validator_without_sample_app_for_bridge" {
+  count  = !var.sample_app_callable && var.ecs_taskdef_network_mode == "bridge" ? 1 : 0
+  source = "../validation"
 
+  validation_config            = var.validation_config
+  region                       = var.region
+  testing_id                   = module.common.testing_id
+  metric_namespace             = "${module.common.otel_service_namespace}/${module.common.otel_service_name}"
+  mocked_server_validating_url = "http://${aws_lb.mocked_server_lb.dns_name}:${module.common.mocked_server_lb_port}/check-data"
+  cloudwatch_context_json = jsonencode({
```

Comment:
Actually I changed the logic here in recent PR

https://github.com/aws-observability/aws-otel-test-framework/blob/acddf135e1205078a2eb720540ed503e084bb751/terraform/ecs/main.tf#L49

 https://github.com/aws-observability/aws-otel-test-framework/blob/acddf135e1205078a2eb720540ed503e084bb751/terraform/ecs/main.tf#L287-L303

https://github.com/aws-observability/aws-otel-test-framework/blob/acddf135e1205078a2eb720540ed503e084bb751/terraform/templates/defaults/cloudwatch_context.json#L2

The default json contains cluster name, can you change to use the default.

Reply:
Changed it thanks!